scripts/import_assets/test_import_assets.py

```diff
@@ -0,0 +1,100 @@
+from unittest import TestCase
+import import_assets
+
+
+class TestMyUnits(TestCase):
+
+    def test_str_to_bool_one(self):
```

Comment:
It would be nice if you could use more meaningful names for tests. One, two, three, four don't really say what the test is checking, and imagine getting test_str_to_bool_five test failing in the report. It would be better to use following schema:

`test_str_to_bool_should_raise_value_error_on_empty_str`.

Reply:
Done!